package.json

```diff
@@ -12,7 +12,7 @@
         "debug": "0.8.0 - 3.5.0"
     },
     "devDependencies": {
-        "@types/node": "^8.0.37",
+        "@types/node": "8.10.30",
```

Comment:
I think, I have fixed everything on my box. Just pushing the commit on another PR.